core/src/main/java/org/axonframework/eventsourcing/eventstore/EmbeddedEventStore.java

```diff
@@ -108,9 +108,37 @@ public EmbeddedEventStore(EventStorageEngine storageEngine, MessageMonitor<? sup
      */
     public EmbeddedEventStore(EventStorageEngine storageEngine, MessageMonitor<? super EventMessage<?>> monitor,
                               int cachedEvents, long fetchDelay, long cleanupDelay, TimeUnit timeUnit) {
+        this(storageEngine, monitor, 10000, 1000L, 10000L, TimeUnit.MILLISECONDS,
```

Comment:
This was probably not intended? : )

Reply:
The ` 10000, 1000L, 10000L,` part that is.

Reply:
Nope, I think so too. I am adjusting this in #915 :+1: